doc/Language/objects.pod6

```diff
@@ -138,7 +135,7 @@ another class.
 =head2 Attributes
 X<|Attribute> X<|Property> X<|Member> X<|Slot>
 
-Attributes are variables that exist per instance of a class. They are where
+Attributes are variables that exist per object basis. They are where
```

Comment:
Yeah. That was horrible. How about:
Attributes are variables that exist per class (are class attributes) or per instance of a class (are instance or object attributes).